test/e2e/runner/main.go

```diff
@@ -116,13 +124,26 @@ func NewCLI() *CLI {
 	cli.root.Flags().BoolVarP(&cli.preserve, "preserve", "p", false,
 		"Preserves the running of the test net after tests are completed")
 
-	cli.root.AddCommand(&cobra.Command{
+	cli.root.Flags().StringVarP(&cli.logLevel, "log-level", "l", config.DefaultPackageLogLevels(),
+		"Sets the log level of the test net")
+
+	setupCmd := &cobra.Command{
 		Use:   "setup",
 		Short: "Generates the testnet directory and configuration",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			logLevel, err := cmd.Flags().GetString("log-level")
+			if err != nil {
+				return err
+			}
+			cli.testnet.LogLevel(logLevel)
 			return Setup(cli.testnet)
 		},
-	})
+	}
+
+	// add log level flag to setupCmd
+	setupCmd.Flags().AddFlag(cli.root.Flags().Lookup("log-level"))
```

Comment:
Hm, this can be a bit confusing. I would expect to be able to use `runner run --log_level debug`, but seems like it only works for setup. In order to reduce confusion, maybe just use manifest for this? Alternatively, we could pass the flag level on to the Docker containers as runtime flags too, since Tendermint supports `--log_level`. But of course, that would replace the per-node levels, so we would need support for per-node levels in the flags.